### PR TITLE
Add the test case for pipeline

### PR DESCRIPTION
**Pull request by intom**
- Add the pipeline test case
 - Handle the exceptions for the existing api

### CODE REVIEW

**Comment by chunseoklee** [comment]
@intom Due to test case policy, we should keep the ratio 50:50 for positive:negative tests. Could you please add more negative tests ?

**Comment by intom** [comment]
> @intom Due to test case policy, we should keep the ratio 50:50 for positive:negative tests. Could you please add more negative tests ?

I see. I will add negative tests

File: tests/nnfw_api/src/ValidationTestPipelineSession.cc [hunk 1]
```diff
@@ -73,6 +69,148 @@ TEST_F(ValidationTestPipelineSession, pipeline_session_test_model)
...
+TEST_F(ValidationTestPipelineSession, prepare_pipeline_002)
+{
+  build_partition_map();
+
+  NNFW_ENSURE_SUCCESS(nnfw_create_session(&_session));
+  NNFW_ENSURE_SUCCESS(nnfw_load_model_from_modelfile(
+    _session, NNPackages::get().getModelAbsoluteFilePath("mobilenet_v1_1.0_224").c_str()));
+  NNFW_ENSURE_SUCCESS(nnfw_prepare_pipeline(_session, "./partition_map.json"));
+
+  remove("./partition_map.json");
```
Comment (hasw7569) [inline tests/nnfw_api/src/ValidationTestPipelineSession.cc:96]: `build_partition_map` also invoke `remove` function.
do we have to invoke this function 2 times in each test ?

Comment (intom) [reply tests/nnfw_api/src/ValidationTestPipelineSession.cc:94]: I missed it. Thank you for your review.
I removed this function.

**Review by chunseoklee — APPROVED** [review]

**Review by hasw7569 — APPROVED** [review]

**Review by juitem — APPROVED** [review]
LGTM

**Pull request merged**